frontend/app/(tabs)/lists/[id].tsx

```diff
@@ -1,21 +1,129 @@
-import { useLocalSearchParams, Stack } from "expo-router";
-import { Button, Text, View } from "react-native";
-import React from "react";
+import { useLocalSearchParams, useRouter } from "expo-router";
+import {
+  Text,
+  FlatList,
+  Button,
+} from "react-native";
+import { View, Card, XStack, Stack } from "tamagui";
+import React, { useEffect, useState } from "react";
 
 export default function Page() {
   const { id } = useLocalSearchParams();
+  const [recipes, setRecipes] = useState([]);
+
+
+  useEffect(() => {
+    if(id) {
+      fetchDataFromBackend();
+    } else {
+      console.log('No id found.');
+    }}, []);
+
+
+  const fetchDataFromBackend = async () => {
+    try {
+      const response = await fetch(`https://open-recipes.onrender.com/recipe-lists/${id}`);
+      if (!response.ok) {
+        throw new Error('Failed to fetch data');
+      }
+
+      const data = await response.json();
+      const {recipes} = data;
+      setRecipes(recipes);
+    } catch (error) {
+      console.error('Error fetching data:', error.message);
+    }
+  };
+
+  
+
   return (
-    <View>
-      <Stack.Screen options={{ headerTitle: "" }} />
-      <Text style={{ fontSize: 20 }}>Recipe #1: ...</Text>
-      <Text></Text>
-      <Text style={{ fontSize: 20 }}>Recipe #2: ...</Text>
-      <Text></Text>
-      <Text style={{ fontSize: 20 }}>Recipe #3: ...</Text>
-      <Text></Text>
-      <Text style={{ fontSize: 20 }}>Recipe #4: ...</Text>
-      <Text></Text>
-      <Text style={{ fontSize: 20 }}>Recipe #5: ...</Text>
+    <View style={{ width: "100%", flex: 1 }}>
+      <View style={{ flex: 1, marginVertical: 20 }}>
+        <View style={{ flex: 1 }}>
+          <RecipeComponent data={recipes} />
```

Comment:
Nice, i like that you're splitting up your code into components. This component should probably be called "RecipesComponent" not the singular version. It is responsible for rendering multiple recipies